src/schism/schism_esmf_util.F90

```diff
@@ -69,8 +69,9 @@ module schism_esmf_util
   end type
 
   type(ESMF_MeshLoc) :: meshloc
+  logical :: dbug
 
-  public meshloc
+  public meshloc, dbug
```

Comment:
better local variable name would be isDebug

Reply:
done